dotnet/src/Connectors/Connectors.Memory.Pinecone/PineconeUtils.cs

```diff
@@ -0,0 +1,261 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Microsoft.SemanticKernel.Connectors.Memory.Pinecone.Http.ApiSchema;
+using Microsoft.SemanticKernel.Connectors.Memory.Pinecone.Model;
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Pinecone;
+
+public static class PineconeUtils
+{
+    /// <summary>
+    ///  The maximum size of the metadata associated with each vector.
+    ///  See https://docs.pinecone.io/docs/metadata-filtering#supported-metadata-size
+    /// </summary>
+    public const int MaxMetadataSize = 40 * 1024;
+
+    /// <summary>
+    ///  The default dimension for Pinecone vectors. Equivalent to text-embeddings-ada-002 dimension.
+    /// </summary>
+    public const int DefaultDimension = 1536;
+
+    public const string DefaultIndexName = "sk-index";
```

Comment:
Since index is synonymous with collection, is this necessary if CreateCollectionAsync is changed to call CreateIndexAsync?

Reply:
@awharrison-28 Responded about this in first comment, please let me know if that makes sense.